drivers/hmis/app/models/hmis/hud/processors/client_processor.rb

```diff
@@ -40,10 +50,9 @@ def information_date(_)
     # TODO: move actual logic here once ClientInputTransformer is removed because we stop using that mutation
     private def race_attributes(attribute_value)
       Types::HmisSchema::Transformers::ClientInputTransformer.multi_field_attrs(
-        attribute_value,
-        Hmis::Hud::Client.race_enum_map,
+        attribute_value, Hmis::Hud::Client.race_enum_map,
         :data_not_collected,
-        :RaceNone,
+        :RaceNone
```

Comment:
This looks like rubocop would complain about it? I thinkwe have a rule for commas at the end of each line